src/test/kotlin/org/rust/ide/annotator/RsStdlibErrorAnnotatorTest.kt

```diff
@@ -24,4 +24,11 @@ class RsStdlibErrorAnnotatorTest : RsAnnotatorTestBase(RsErrorAnnotator::class.j
 
         <error descr="Can't find crate for `litarvan` [E0463]">extern crate litarvan;</error>
     """)
+
+    fun `test impl Drop for Trait E0120`() = checkErrors("""
```

Comment:
See https://github.com/intellij-rust/intellij-rust/pull/3789#discussion_r281042654 to move this test to `RsErrorAnnotatorTest.kt`